nucliadb_telemetry/nucliadb_telemetry/tests/integration/test_fastapi.py

```diff
@@ -210,7 +180,7 @@ class TestCasePrometheusMiddlewareFilterUnhandledPaths:
     @pytest.fixture(scope="class")
     def app(self):
         app_ = Starlette()
-        app_.add_middleware(PrometheusMiddleware, filter_unhandled_paths=True)
```

Comment:
IIRC I added `filter_unhandled_paths` so that urls that do not match any of our endpoints do not get into prometheus metrics.

Without it, if someone would do a GET `api/kb/<kbid>/non-existing-endpoint`, the <kbid> was appearing in the metric labels

Reply:
Yup, totally makes sense. I just make this the default behavior now and not optional.